<a href="https://colab.research.google.com/github/joaozz21/Python-ex/blob/main/Lista01.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

1.  Escreva um programa que ajude uma loja a fazer uma venda cujo pagamento  e parcelado. Assuma que a loja divide o preco em 19 parcelas iguais e sem quaisquer acrescimos (juros, multas, etc). Faca um programa que:
- leia do teclado o preco do produto;
- imprima quanto custara cada parcela (arredondando com duas casas decimais).
- Veja aqui um exemplo:

  ## Entrada:
  665.95
  ## Saida:
  35.05

In [18]:
preco=float(input())
parcelas=preco/19
print(f'{parcelas:.2f}')

3.600
3.300
1.6498744056784245


2.  Escreva um programa para ler (do teclado) dois numeros e somar o primeiro pelo segundo.
Mostrar o resultado com duas casas decimais.

  Veja aqui um exemplo:
  ## Entradas:
  41
  ## Saida:Resultado =
  5.00

In [ ]:
num1=float(input())
num2=float(input())
soma=num1+num2
print(f'Resultado = {soma:.2f}')

3.  Faca um programa contendo instrucoes para ler do teclado um numero inteiro que representa uma quantidade x de segundos. O programa deve imprimir na tela o numero de horas, minutos e segundos contidos em x no formato HH:MM:SS. Por exemplo, se x= 5000, entao o resultado devera ser 01:23:20, uma vez que 5000 segundos correspondem a 1 hora, 23 minutos e 20segundos.  Se x= 3600, o resultado devera ser 01:00:00(exatamente uma hora).
Um exemplo e mostrado a seguir.
  ## Entrada lida com o comando input:
  3600
  ## Saida correspondente a entrada:
  01:00:00

In [24]:
x=int(input())
s= x%60
m= (x//60)%60
h= x//3600
print(f"{h:02d}:{m:02d}:{s:02d}")

7200
02:00:00


4.  Faca um programa para calcular e imprimir o valor da expressao (formula) abaixo, a partir das variaveis reais A e B.  As variaveis A e B devem ser lidas do teclado, nessa ordem.

   G=1−exp(−A/B)/2∗B

   Mostrar o resultado, com tres casas decimais, conforme exemplo a seguir:

   ## Entradas:
   3.600

   3.300
   ## Saida:
   G = 0.101

In [31]:
import math

A=float(input())
B=float(input())
G=(1-math.exp(-A/B))/(2*B)

print(f"G = {G:.3f}")

3.600
3.300
G = 0.101
